src/index.html

```diff
@@ -3,10 +3,379 @@
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-amount">$2,199</div>
+        </div>
+
+        <a
+          href="#buy"
+          class="card__button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__image"
+          src="/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27&#34; Retina 5K Monoblock"
+        >
+
+        <div class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+        <div class="card__code">Product code: 195434</div>
+
+        <div class="card__mark">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-amount">$2,199</div>
+        </div>
+
+        <a
+          href="#buy"
+          class="card__button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__image"
+          src="/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27&#34; Retina 5K Monoblock"
+        >
+
+        <div class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+        <div class="card__code">Product code: 195434</div>
+
+        <div class="card__mark">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-amount">$2,199</div>
+        </div>
+
+        <a
+          href="#buy"
+          class="card__button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__image"
+          src="/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27&#34; Retina 5K Monoblock"
+        >
+
+        <div class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
```

Comment:
better to use semantic tags here better will be h2, with another text wrappers better to use paragrapth